src/Carbon/Translator.php

```diff
@@ -107,6 +107,16 @@ public function removeDirectory(string $directory)
         }));
     }
 
+    /**
+     * Returns the translation.
+     *
+     * @param int    $id
+     * @param array  $directory
```

Comment:
There is no directory, this is the interpolations parameter, we named if `parameters` because it's the same name as in symfony/translation used by the parent method.

Reply:
Updated it, sorry about that.